acquire/outputs/base.py

```diff
@@ -96,7 +96,7 @@ def write_volatile(
             buf = b""
             size = 0
 
-        self.write_bytes(output_path, buf, entry, size)
+        self.write_bytes(output_path, buf, entry, size=size)
```

Comment:
Because that is what the call signature of `write_bytes()` is. And it is bad practice to pass named arguments as positionals. I noticed `entry` was not yet passed as a named argument, so I changed that.